lib/cockpit-components-select.jsx

```diff
@@ -39,16 +39,24 @@ define([
             onChange: React.PropTypes.func,
             id: React.PropTypes.string,
         },
+        handleDocumentClick: function(ev) {
+            // clicking outside the select control should ensure it's closed
+            if (!React.findDOMNode(this).contains(ev.target))
```

Comment:
> Is this really correct here, when invoked from addEventListener or removeEventListener callbacks? Are you relying on the this from the outer scope? If so, I would call it self.

Well, it works. I could optimize by storing the result of `React.findDOMNode(this)` in componentDidMount and binding that to the function. What do you think?